modules/rpc/handler.go

```diff
@@ -88,3 +90,27 @@ func authorize(ctx context.Context, host service.Host) (context.Context, error)
 	}
 	return ctx, nil
 }
+
+type panicInboundMiddleware struct{}
```

Comment:
this follows the naming convention with `contextInboundMiddleware` and `authInboundMiddleware` right now
maybe change all of them in another diff?